plugins/hybrid_analysis/requirements.txt

```diff
@@ -1,3 +1,4 @@
 # List third-party dependencies here, separated by newlines. 
 # All dependencies must be version-pinned, eg. requests==1.2.0
-# See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
\ No newline at end of file
+# See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
+validators==0.18.2
```

Comment:
can you please explain why the validators version is pinned

Reply:
Yes i could explain this is the external library, so for external libraries we always must provide the version of library. If you are interested here is the documentation for the library https://validators.readthedocs.io/en/latest/